client/src/components/Post.jsx

```diff
@@ -164,7 +152,7 @@ class Post extends React.Component {
                 </div>
               </div>
             </div>
-          </div>
+          </div>*/}
```

Comment:
If we don't need this code, should we take it out?

Reply:
I left this in since it was the code to add comments that the karms_facebook group added. It adds the fake, hard-coded Fred comments. I think we decided that if we wanted to hide placeholder fake content, we wouldn't delete it, but just comment out.

Reply:
cool.